src/Estimators/EstimatorManagerNew.h

```diff
@@ -134,6 +147,8 @@ class EstimatorManagerNew
   std::size_t getNumScalarEstimators() { return scalar_ests_.size(); }
 
 private:
+  EstimatorManagerInput input_;
```

Comment:
Delete? I saw all its content gone after constructing all the individual estimators.

Reply:
Makes sense we can return if we ever actually create any estimators scope attributes or parameters.